common/djangoapps/student/models.py

```diff
@@ -1028,7 +1028,7 @@ class CourseEnrollmentManager(models.Manager):
     Custom manager for CourseEnrollment with Table-level filter methods.
     """
 
-    def num_enrolled_in(self, course_id):
+    def num_enrolled_in(self, course_id, max_enrollment_for_buttons):
```

Comment:
`max_enrollment_for_buttons` -> `max_enrollments`
Should have a more generic variable name!

Reply:
Okay I'd change it.